src/sentry/static/sentry/app/components/hovercard.jsx

```diff
@@ -204,10 +223,6 @@ const StyledHovercard = styled('div')`
   ${p => (p.placement === 'right' ? 'margin-left: 15px' : '')};
 `;
 
-const Container = styled('span')`
```

Comment:
Why did this go away?

Reply:
do you know why you'd put it there? it was messing with the new hovercard because we can't position `children` w.r.t. to an ancestor of `Hovercard` anymore. neither @markstory nor I could figure out why it was there to begin with so I removed it (will do visual regression testing to make sure it doesn't break existing hovercards)

Reply:
tested a few existing hovercards and they're fine